sdk/storage/Azure.Storage.Blobs.Cryptography/src/Models/EncryptionAgent.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.Serialization;
+
+namespace Azure.Storage.Blobs.Specialized.Cryptography.Models
+{
+    /// <summary>
+    /// Represents the encryption agent stored on the service.
+    /// </summary>
+    [DataContract]
+    public struct EncryptionAgent
```

Comment:
Should this be public?  Isn't there only one accepted value for this?

Reply:
It's being serialized. The serialization package needs to be able to see the class.